### DISCUSSION TITLE
Artifact auto updgrade

### DISCUSSION THREAD

**Discussion by ElvinFox** [FarbstoffRSL/RSL-Helper#392] [Ideas]
It would be great if we could have auto upgrade option in addition to inventory sell:
- select specific items or at least visible ones and make them +4/+8/+12/+16 (to see where stats roll)
This could greatly improve the process of inventory management !

**Comment by RedMan1911** [comment]
> It would be great if we could have auto upgrade option in addition to inventory sell:
> 
> * select specific items or at least visible ones and make them +4/+8/+12/+16 (to see where stats roll)
>   This could greatly improve the process of inventory management !

I had the same idea in my mind during the Artifact Enhancement event. As a budget solution, I am upgrading all 6* and 5* to 8lvl to see the rolls. So would be great if it could be automatized anyhow.
@ElvinFox maybe you should change the summary to Artifact auto updgrade. :)

**Comment by RedMan1911** [comment]
@ElvinFox , check the answer here: https://github.com/FarbstoffRSL/RSL-Helper/issues/402
